panel/models/card.ts

```diff
@@ -51,7 +51,7 @@ export class CardView extends ColumnView {
 
     const {background, button_css_classes, header_color, header_tag, header_css_classes} = this.model
 
-    this.el.style.backgroundColor = background != null ? background : ""
+    this.el.style.backgroundColor = (background != null ? background : "" as any)
```

Comment:
Use `color2css()` from `core/util/color`.

Reply:
In general one can't assume anymore that a color is a string, or even if it's actually a string, that it's a CSS color spec or is a widely supported CSS color spec (e.g. we allow `#RRGGBBAA` syntax, which isn't widely supported yet).

Reply:
Thanks!